change/@ni-nimble-components-8010f392-806a-4399-bf80-9fcd41f97cb5.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
When I open the [built storybook](https://60e89457a987cf003efc0a5b-dalmornnki.chromatic.com/?path=/story/wafermap--wafer-map&args=colorScale:set1) I'm seeing that the die labels are pixelated/blurry. Is this expected?

<img width="838" alt="image" src="https://user-images.githubusercontent.com/10500124/210461866-461fbb57-3800-4da0-84ad-1b84c5538249.png">

Reply:
The issue seems to have gone away after setting a default size for the nimble-wafermap component